dynacred/src/Data.ts

```diff
@@ -357,7 +360,7 @@ export class Data {
     async connectByzcoin(): Promise<ByzCoinRPC> {
```

Comment:
This should either take a `ByzCoinRPC = null` or a `Config = null`. Personally I tend to put everything as soon as possible into the most interpreted way possible, so I tend to propose the following:
`async connectByzcoin(bc: ByzCoinRPC = null)`
and then
```
if (this.bc == null && bc == null){
  throw new Error("No ByzCoin definition found")
}
```